src/irgen.jl

```diff
@@ -1,8 +1,12 @@
 # LLVM IR generation
 
-function irgen(@nospecialize(job::CompilerJob), method_instance::Core.MethodInstance)
+function irgen(@nospecialize(job::CompilerJob), method_instance::Core.MethodInstance; specfunc_entry=true)
     mod, compiled = @timeit_debug to "emission" compile_method_instance(job, method_instance)
-    entry_fn = compiled[method_instance].specfunc
+    if specfunc_entry then
```

Comment:
```suggestion
    if specfunc_entry
```

Too much shell :smile: